tests/spec/core/utils-spec.js

```diff
@@ -307,22 +307,26 @@ describe("Core - Utils", () => {
     const localizationStrings = {
       en: { foo: "EN Foo", bar: "EN Bar" },
       ko: { foo: "KO Foo" },
+      de: { foo: "DE Foo" },
     };
 
     it("returns localized string in given language", () => {
-      const intl = getIntlData(localizationStrings, "ko");
-      expect(intl.foo).toBe("KO Foo");
+      const intlKo = getIntlData(localizationStrings, "ko");
+      expect(intlKo.foo).toBe("KO Foo");
+
+      const intlDe = getIntlData(localizationStrings, "de");
+      expect(intlDe.foo).toBe("DE Foo");
 
       const intlEn = getIntlData(localizationStrings, "en");
       expect(intlEn.foo).toBe("EN Foo");
     });
 
-    it("falls back to English string if key does not exit in language", () => {
+    it("falls back to English string if key does not exist in language", () => {
       const intl = getIntlData(localizationStrings, "ko");
       expect(intl.bar).toBe("EN Bar");
     });
 
-    it("falls back to English string if language does not exit in language data", () => {
+    it("falls back to English string if key does not exist in language", () => {
```

Comment:
Probably should be merged into the former test.

Reply:
This test is important. It checks if entire language doesn't exist in `localizationStrings`. The other one tests if language exists but given key doesn't exist in that language.

Reply:
cc: @sebastian-meyer

Reply:
I fixed the typo in https://github.com/w3c/respec/pull/2687/commits/6ade5f0826653470e380a40d819e3f5a9bbc5e9b and removed the duplicate test in https://github.com/w3c/respec/pull/2687/commits/e1d3b05fc02396daa15622317067b6fc0e16ffdf

Reply:
Oh, I see. I think "language data" should be renamed as "localization data" BTW.

Reply:
Changed the test's title in https://github.com/w3c/respec/pull/2687/commits/6052246b27ef5d95064a10e2e1c51ed1d2ec5c95